storage/badger/receipts.go

```diff
@@ -188,28 +188,55 @@ func (r *ExecutionReceipts) ByBlockID(blockID flow.Identifier) (*flow.ExecutionR
 }
 
 func (r *ExecutionReceipts) ByBlockIDAllExecutionReceipts(blockID flow.Identifier) ([]*flow.ExecutionReceipt, error) {
```

Comment:
I'm a bit uncomfortable making this change at the storage layer, since this method id used by consensus nodes as well, https://github.com/onflow/flow-go/blob/fc386f88fd2b075ee171d996d9a4e4445b92c726/engine/consensus/matching/engine.go#L807-L823

I'd rather we use the same approach we used in the consensus node hotfix

Reply:
ok will swtich to using that in the access node code.

Reply:
moved it to the access node code